src/components/Layout/index.module.css

```diff
@@ -1,3 +1,4 @@
 .root {
+  padding: 0 28px 102px;
```

Comment:
Please, write these out instead of using a shorthand. I prefer to see in one go what's happening. This way, you also don't need to reset spacings to `0` out of convenience, and you can leave those out completely.

```
padding-right: 28px;
padding-bottom: 102px;
padding-left: 28px;
```